component-runtime-impl/src/main/java/org/talend/sdk/component/runtime/record/MappingUtils.java

```diff
@@ -65,11 +65,6 @@ public static <T> Object coerce(final Class<T> expectedType, final Object value,
             }
         }
 
-        // we store decimal by string for AvroRecord case
-        if ((expectedType == BigDecimal.class) && String.class.isInstance(value)) {
```

Comment:
Hm. If it was already supported by the line 212 in mapString, why did we have this if statement?

Reply:
It passed the review. Reviewer did not catch it as mapString method wasn't modified I think (and it's maybe me :-D )